### DISCUSSION TITLE
钱包对接，完成授权后无法自动返回，望给予技术支持

### DISCUSSION THREAD

**Discussion by btok1234** [consenlabs/webview#58] [Q&A] [answered]
您好！这边想向您反馈个问题。我们尝试使用WC协议将我们的app与你们的钱包进行对接（我们使用的是walletconnect官方sdk原生接入方式，并非dapp方式），但是，完成授权签名“操作后，停留在你们的钱包应用页面，无法自动回到我们的应用情况，需要手动操作返回。针对这种无法自动返回的情况，恳请您们协助支持或者对这个问题进行优化。感谢您的的帮助！

**Comment by xwartz** [comment, marked as answer]
imToken 目前不支持通过 deeplink 跳转回原 App
